doc/source/contributing/how_to_contribute.rst

```diff
@@ -3,62 +3,70 @@
 How to Contribute
 =================
 
-On this page we briefly provide basic instructions for contributing to icepyx.
+On this page we provide brief instructions for contributing to icepyx.
 We encourage users to follow the `git pull request workflow <https://www.asmeurer.com/git-workflow/>`_.
-For contribution examples, please see contribution guidelines.
+For contribution examples, please see :doc:`contribution guidelines <contribution_guidelines>`.
 
 
 Contributing for the first time
 -------------------------------
 1. If you don't have one, sign up for a GitHub account (visit https://github.com/ and ‘sign up for GitHub account’).
 
 2. Clone the icepyx repo: Open a terminal window.
-Navigate to the folder on your computer where you want to store icepyx.
-For example,
+   Navigate to the folder on your computer where you want to store icepyx.
+   For example,
```

Comment:
This text and code blocks, because they weren't indented, were not rendered as "part of" the numbered list element. Indenting them puts them inside.

Reply:
thanks for the explanation and cleaning up this formatting!